sematic/ui/src/components/Notes.tsx

```diff
@@ -1,5 +1,10 @@
-import { Box, Typography, useTheme } from "@mui/material";
+import { ContentCopy } from "@mui/icons-material";
+import { Box, ButtonBase, Tooltip, Typography, useTheme } from "@mui/material";
+import { useCallback, useContext } from "react";
+import { redirect, useNavigate } from "react-router-dom";
```

Comment:
`$ npm install` because of the version change in `packages.json`.
But I should actually remove this as I was not able to make it work. Will clean up.